app/lib/efile/nj/nj1040_calculator.rb

```diff
@@ -120,18 +120,18 @@ def calculate_property_tax_deduction
           return nil
```

Comment:
I guess using line_or_zero wouldn't work for line 119 but could use something like `@lines[:NJ1040_LINE_40A]&.value`